tests/acceptance/utils.js

```diff
@@ -1,13 +1,34 @@
-import { ClientFunction } from 'testcafe';
+import { ClientFunction, t } from 'testcafe';
+import IE11NoCacheHook from './ie11nocachehook';
 
 /* global location */
 
 export async function browserRefreshPage () {
   await ClientFunction(() => location.reload())();
-  await new Promise(resolve => setTimeout(resolve, 2500));
+  return t.wait(2500);
```

Comment:
For the refresh, back, and forward actions, could we use `t.navigateTo`? The `navigateTo` has a built-in waiting mechanism. From my experience with Selenium, it is better to rely on the built-in mechanisms than to add explicit `wait`s. Adding a `wait` like this can lead to spurious failures.

Reply:
I tried it and I think navigateTo is functionally different than a refresh. Is it alright if I keep the waits for now, since they existed previously? I tried removing them and it looks like that makes things fail more often than not. There's probably some way to ask testcafe to smartly wait for things but it seems a little out of scope